src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
I don't think I agree on this aspect if we are supporting the IDEs. There's a reason why IntelliJ (and other IDEs) made their inspection suppressions as comments, so they become just comments outside of the IDE.

Being IDE-agnostic does not mean we should throw away good features that an IDE offer unless they specifically conflict with the workflow. IntelliJ has taken the efforts to specifically make suppressing IntelliJ specific inspections comments so it does not interfere with the workflow.

As for whether we treat the default settings as authoritative, that's up to the project? We should turn those inspections off completely if we don't agree with it, not ignore them and leave the warnings hanging around.

As for this particular instance, I think IntelliJ's defaults are sane here. There is a general consensus that it is a good thing to add generic font families and it is also recommended by various sources and specifications:
>  Authors are encouraged to append a generic font family as a last alternative for improved robustness.

https://drafts.csswg.org/css-fonts-4/#font-family-prop
https://www.w3.org/TR/css-fonts-3/#font-family-prop

> Note: You should always include at least one generic family name in a font-family list, since there's no guarantee that any given font is available. This lets the browser select an acceptable fallback font when necessary.
https://developer.mozilla.org/en/docs/Web/CSS/font-family


So as this is a special case where it does not make sense that an icon font have generic fallback, I think suppressing the warning is the way to go.